.env

```diff
@@ -1,7 +1,8 @@
-﻿COMPOSE_PROJECT_NAME=lexbox
-LEXBOX_POSTGRES_PW=972b722e63f549938d07bd8c4ee5086c
-LEXBOX_HASURA_SECRET_KEY=c67eeb626bee405d883b482046934860
```

Comment:
you're correct there will always be lexbox stuff, but my reason was to indicate ownership, that this was something we defined and owned, instead of something specified by a 3rd party. But this is fine the way that it is.